pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -99,7 +99,10 @@
         "controller.minion.instances.cleanup.task.frequencyInSeconds";
     public static final String MINION_INSTANCES_CLEANUP_TASK_INITIAL_DELAY_SECONDS =
         "controller.minion.instances.cleanup.task.initialDelaySeconds";
-    public static final String TASK_METRICS_EMITTER_FREQUENCY_IN_SECONDS = "controller.minion.task.metrics.emitter.frequencyInSeconds";
+    public static final String MINION_INSTANCES_CLEANUP_TASK_MAX_OFFLINE_TIME_RANGE_SECONDS =
+        "controller.minion.instances.cleanup.task.maxOfflineTimeRangeSeconds";
```

Comment:
```suggestion
        "controller.minion.instances.cleanup.task.maxOfflineTimeBeforeDelete";
```
And change the variable name accordingly